### DISCUSSION TITLE
Best calс mode for small details

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#169] [Q&A]
I have modelA with good body shapes and modelB with realistic skin textures.
I want to get skin textures from B to A with the minimal impact on A. I understand that I have to use elemental merging and I've done some tests but the results are unclear for me.

What is the best calculation mode for copying small details?

**Comment by hako-mikan** [comment]
I want to know. I believe it's unavoidable that other elements will mix in, but we have no choice but to merge only the strong layer related to skin-related elements. If other elements mix even with layer separation, then we have no choice but to search for effective elements in Elemetal. Unfortunately, I don't possess information about skin-related aspects. I think the Adjust function is effective regarding texture.

**Reply by miasik** [reply]
Sure, only elemental merging can help me. I've corrected the opening post. The question is — which calc mode is better for small details?

**Reply by hako-mikan** [reply]
I'm not familiar with the differences in calc mode. Most of my time is spent on development, and I haven't been able to do much merging work. I think the model creators would know more about the details of merging.
